### PR TITLE
Confirmation page edits

### PR DESCRIPTION
**Pull request by hbillings**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/387)

## What does this change?
Fixes a bunch of spacing, color, and type issues with the confirmation page. 

## Screenshots (for front-end PR):
Note that this does NOT touch any of the elements that are shared with Review, as I believe @cantsin's PR will fix that part of this page.
Screenshot_2020-05-12 Civil Rights Division - Department of Justice

### Author

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [x] Check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/cts_forms/templates/forms/confirmation.html [hunk 1]
```diff
@@ -34,8 +34,8 @@ <h2>
       </div>
     </div>
       <div class="crt-success-card__content">
-          <h3>{% trans "Please save this page for your records" %}</h3>
-          <p>{% trans "Write down your record number and save or print a copy of this page for personal records." %}</p>
+          <h3 class="intake-section-title">{% trans "Please save your record number for tracking." %}</h3>
+          <p>{% trans "If you want a copy of this report for your personal records, save or print a copy of this page." %}</p>
```
Comment (LindsayYoung) [inline crt_portal/cts_forms/templates/forms/confirmation.html:38]: Nice- this will make it into the translation if I can merge it now

**Comment by LindsayYoung** [comment]
I am seeing the links in the footer as dark blue
Screen Shot 2020-05-12 at 5 00 43 PM

File: crt_portal/cts_forms/templates/forms/confirmation.html (hunk 1 above)
Comment (LindsayYoung) [reply crt_portal/cts_forms/templates/forms/confirmation.html:38]: I can also grab this for translation, depending on if this needs more time

**Comment by hbillings** [comment]
Thank you for catching both of those! I can push a fix in just a minute

**Comment by LindsayYoung** [comment]
The information cards for the review page and the confirmation page are looking different. The mock said they should look the same https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/387

Locally, I am going to look at what it looks like when #462 is merged

**Comment by hbillings** [comment]
@LindsayYoung Okay, the links and the font size of the record number should be fixed. 

Thanks for checking it out locally! What I could tell from the code made it seem like that part of the page should be okay once those PRs are merged, so I didn't want to step on James' toes, but I'm happy to fix any stray outliers that seem weird once his PRs are in!

**Comment by LindsayYoung** [comment]
Thanks for the quick fixes! I think for tonight, we can get the #460 and #462 merged, first- sounds like you are up for it. I will grab the text so it make is to translation. 

We will have to untangle the data card changes, there is some regrouping for the categories like address and we want them to be similar on the confirmation and submission pages. 

Thank you!

File: crt_portal/cts_forms/templates/forms/grouped_questions.html [hunk 2]
```diff
@@ -6,10 +6,8 @@
       {{ question_group.group_name }}{% if not question_group.optional %}<span class="field-required--group">{% trans "required" %}</span>{% endif %}
     </legend>
     {% if question_group.help_text %}
-      <p class="margin-bottom-4 margin-top-0 {{question_group.label_cls}}" id="{{question_group.ally_id}}">
-        <em class="help-text">
-          {{ question_group.help_text }}
-        </em>
+      <p class="help-text {% if question_group.label_cls %}{{question_group.label_cls}}{% endif %}" id="{{question_group.ally_id}}">
```
Comment (cantsin) [inline crt_portal/cts_forms/templates/forms/grouped_questions.html:9]: :heart_eyes_cat:

**Comment by LindsayYoung** [comment]
With @cantsin review and all the merge conflicts resolve Let's get this into the release 🎉

**Pull request merged**